tests/post-command.bats

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bats
+
+load '/usr/local/lib/bats/load.bash'
+
+post_command_hook="$PWD/hooks/post-command"
+prefix="BUILDKITE_PLUGIN_AWS_PUT_EVENT"
+prefix_entries="BUILDKITE_PLUGIN_AWS_PUT_EVENT_ENTRIES"
+
+function aws() {
+	echo ${SECRET_VALUE:-secret}
```

Comment:
You can probably echo anything here because the value's not used in your tests. Having SECRET_VALUE makes it look like that's part of the plugin behaviour